components/layout/FeatureScopeMarker.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <div class="audit-logs mb-3">
+        <p class="title">Available on:</p>
+        <div class="editions">
+            <div class="edition-item" v-for="edition of editions" :style="`background-color: ${edition.color}`">
+                <p>
+                    {{edition.title}}
+                </p>
+            </div>
+        </div>
+        <div class="version" :style="`background-color: ${versionColor}`">
+            <p>
+                {{ version }}
+            </p>
+        </div>
+    </div>
+</template>
+
+<script>
+    import Section from '../layout/Section.vue';
+
+    export default {
+        components: {Section},
+        props: {
+          editions: {
+                type: Array,
+                required: true,
+          },
+          version: {
+                type: String,
+                required: true
+          },
+          versionColor: {
+            type: String,
+            required: true
+          }
+        },
+    }
+</script>
+
+<style lang="scss" scoped>
+    @import "../../assets/styles/variable";
+
+    .audit-logs {
```

Comment:
can you explain what is this for? the Audit logs part of the GitHub issue was just an example usage in markdown files